docs/tutorials/timeseries/forecasting-quickstart.md

```diff
@@ -1,208 +1,253 @@
+python -m pip install mxnet~=1.9
 ```
 
-or, if you are using a GPU with a matching MXNet package for your CUDA driver. See the 
+If you want to use a GPU, install the version of MXNet that matches your CUDA version. See the
 MXNet [documentation](https://mxnet.apache.org/versions/1.9.1/get_started?) for more info.
 
 ---
+## Loading time series data as a `TimeSeriesDataFrame`
 
-This tutorial demonstrates how to quickly start using AutoGluon to produce forecasts of COVID-19 cases in a 
-country given [historical data from each country](https://www.kaggle.com/c/covid19-global-forecasting-week-4). 
-
-`autogluon.timeseries` provides the `TimeSeriesPredictor` and `TimeSeriesDataFrame` classes for interacting 
-with time series models. `TimeSeriesDataFrame` contains time series data. The `TimeSeriesPredictor` class 
-provides the interface for fitting, tuning and selecting forecasting models.
-
-
+First, we make several imports
 ```{.python .input}
 import pandas as pd
 from matplotlib import pyplot as plt
 
-from autogluon.timeseries import TimeSeriesPredictor, TimeSeriesDataFrame 
+from autogluon.timeseries import TimeSeriesDataFrame, TimeSeriesPredictor
 ```
+To use `autogluon.timeseries`, we will only need the following two classes:
 
-`TimeSeriesDataFrame` objects hold time series data, often with multiple "items" (time series) such as different
-products in demand forecasting. This setting is also sometimes referred to as a "panel" of time series.
-`TimeSeriesDataFrame` inherits from 
-[Pandas DataFrame](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.html), and 
-the attributes and methods of `pandas.DataFrame`s are available in `TimeSeriesDataFrame`s.
+- `TimeSeriesDataFrame` stores a dataset consisting of multiple time series.
+- `TimeSeriesPredictor` takes care of fitting, tuning and selecting the best forecasting models.
 
-In our example, we work with COVID case data as of April 2020 where our goal is to forecast the number of confirmed COVID cases
-for each country in the data set.
-Below, we load time series data from an [AWS S3 bucket](https://aws.amazon.com/s3/), and prepare it for use in
-`autogluon.timeseries`. 
-Note that we make sure the date field is parsed by pandas, and provide the columns containing
-the item (`id_column`) and timestamps (`timestamp_column`) to `TimeSeriesDataFrame`. 
-We also plot the trajectories of COVID cases with two example countries:
-Germany and the UK.
 
+In this tutorial we work with COVID case data as of April 2020.
+Our goal is to forecast the cumulative number of confirmed COVID cases for each country in the dataset given the past observations.
 
+We load the dataset from an [AWS S3 bucket](https://aws.amazon.com/s3/) as a `pandas.DataFrame`
 ```{.python .input}
 df = pd.read_csv(
     "https://autogluon.s3-us-west-2.amazonaws.com/datasets/CovidTimeSeries/train.csv",
-    parse_dates=["Date"],
+    parse_dates=["Date"],  # make sure that pandas parses the dates
```

Comment:
Is this really necessary? Shouldn't we be able to convert to the correct type automatically later on without having the user do so during data load? Why not load as a TimeSeriesDataFrame or TabularDataset directly?

Reply:
Currently the API provides options to construct a `TimeSeriesDataFrame` from a pandas DataFrame or from a list of dictionaries. I agree that it would make sense to automatically load from disk, similar to a `TabularDataset`. Will discuss this with Caner and see if it's feasible to do this by 0.6.0.